app/support/auto_expire_reservation.rb

```diff
@@ -14,11 +14,15 @@ def order_details
     purchased_active_order_details
   end
 
+  def earliest_allowed_time
+    Settings.reservations.timeout_period.seconds.ago
```

Comment:
I thought so, but I was getting errors about `Fixnum#ago` not existing. I could dig further